parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -233,6 +240,10 @@ try
         $global:CSEScriptsPackageUrl = $global:CSEScriptsPackageUrl + $WindowsCSEScriptsPackage
         Write-Log "CSEScriptsPackageUrl is set to $global:CSEScriptsPackageUrl"
     }
+
+    Write-Log "calvin: cuda gpu url is set to $global:GpuDriverCudaUrl"
+    Write-Log "calvin: grid gpu url is set to $global:GpuDriverGridUrl"

```

Comment:
remove alias in the logs

Reply:
This is just for testing purpose. I will remove the alias and move the logs inside the new ps1 script.